coordinator/src/routes.rs

```diff
@@ -260,6 +264,32 @@ pub async fn post_trade(
     Ok(invoice.to_string())
 }
 
+#[instrument(skip_all, err(Debug))]
+#[autometrics]
+pub async fn rollover(
```

Comment:
❓Are we certain that we want to trigger rollover by calling an endpoint on the coordinator? I'm not sure, but I'm not sure it is wrong either.

Reply:
I proposed the flow like this https://github.com/get10101/10101/issues/977#issuecomment-1682080659. Based on the learnings from closing expired positions, I think its best to let the app trigger this action.

Reply:
I think your reasoning makes sense, but it's weird to have to grant permission to keep your position alive when we are offering perpetual swaps.

It's unfortunate because I think this is mostly motivated by implementation details (brittle protocol and frequent disconnects). We can roll with this for now, but we might want to revisit this down the line.